### PR TITLE
[AR-2594] Set logatron site

### PR DESCRIPTION
**Pull request by djtreder**
The system currently does not have a unified way of passing the current site around, but all the different messages are defined in one spot, which are all covered by this PR.

### CODE REVIEW

File: lib/mercury/mercury.rb [hunk 1]
```diff
@@ -216,9 +216,18 @@ def dispatch_publisher_confirm(basic_ack)
...
+    Logatron.site = get_message_site(msg.content)
     msg
   end
 
+  def get_message_site(content)
+    content['organization'] ||
+      content['customer'] ||
+      content['site'] ||
+      content.fetch('request', {})['customer'] ||
+      '-'
```
Comment (jwliechty) [inline lib/mercury/mercury.rb:228]: I propose we create a case to standardize the fields in our messages so we don't have this variability.

**Review by jwliechty — APPROVED** [review]
I only lightly reviewed this PR. My expectation is that @wintonpc will scrutinize to the proper degree.

File: lib/mercury/mercury.rb (hunk 1 above)
Comment (jwliechty) [reply lib/mercury/mercury.rb:228]: Actually, is this a mercury concern? Maybe RabbitStew should pass the site to mercury somehow.

Comment (wintonpc) [reply lib/mercury/mercury.rb:228]: This is a rabbit stew concern, not mercury. It would be a mercury concern if it had to be passed along like msg_id is.

Comment (djtreder) [reply lib/mercury/mercury.rb:228]: Agreed; I moved the code to set the Logatron site into Rabbit Stew.

**Pull request closed without merging**